spec/mongo/operation/write/command/insert_spec.rb

```diff
@@ -56,6 +56,31 @@
     end
   end
 
+  describe 'write concern' do
+
+    context 'when write concern is not specified' do
+
+      let(:spec) do
+        { :documents     => documents,
+          :db_name       => db_name,
+          :coll_name     => coll_name,
+          :ordered       => true
+        }
+      end
+
+      it 'does not include write concern in the selector' do
+        expect(op.send(:selector)[:writeConcern]).to be_nil
```

Comment:
That's correct. If the user didn't specify anything, don't send anything. Sending anything has the potential to overwrite the server's default write concern. If the user did specify anything (even what we think of as the default ({w: 1}), you must send it along. 

If you are allowing your users to specify both nil and {}, then I'd take both of these to mean the exact same thing, the server's default.